lib/Wikia/src/Service/User/Permissions/PermissionsService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+interface PermissionsService {
+	public function getGlobalGroups();
```

Comment:
would be quite helpful to have phpdocs describing parameter types and return values